cmd/snap/cmd_snap_op.go

```diff
@@ -277,6 +288,9 @@ func showDone(cli *client.Client, names []string, op string, esc *escapes) error
 				// TRANSLATORS: the args are a snap name optionally followed by a channel, then a version (e.g. "some-snap (beta) 1.3 installed")
 				fmt.Fprintf(Stdout, i18n.G("%s%s %s installed\n"), snap.Name, channelStr, snap.Version)
 			}
+			if !isSnapInPath() {
```

Comment:
Wonder if it'd be better if the message was printed before the `<foo> .. installed`.